reflex/app.py

```diff
@@ -108,6 +108,7 @@ def __init__(self, *args, **kwargs):
         """
         super().__init__(*args, **kwargs)
         state_subclasses = State.__subclasses__()
+        inferred_state = state_subclasses[-1]
```

Comment:
Why do we index `-1` here? Is it because of the `DefaultState` class? I think we should do index `1` in that case. Also, we should throw an exception if `len(state_subclasses) > 1` (or 2 if that's the expected)

Reply:
Nvm, we have the multiple subclasses check right below, missed that